### PR TITLE
Create getting started flow for Sensu 5.2

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- References Sensu Enterprise in the Sensu Go landing page
- Adds an "add a license" step to the main install guide
- Adds a "Try Sensu Enterprise" section to the main getting started guide
- Creates a getting started guide for Sensu Enterprise
- Updates the FAQ

### CODE REVIEW

File: content/sensu-go/5.2/_index.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-<iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-go&type=star&count=true" frameborder="0" scrolling="0" width="87px" height="20px"></iframe> | [Share your feedback][2]
+<iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-go&type=star&count=true" frameborder="0" scrolling="0" width="87px" height="20px"></iframe> | [Learn about Sensu Enterprise][28]
 
 Sensu is the open source monitoring event pipeline built to reduce operator burden and make developers and business owners happy.
 Started in 2011, Sensu’s flexible approach solves the challenges of monitoring hybrid-cloud and ephemeral infrastructures with scalable, automated workflows and integrations with tools you already use.
 **Get started now and feel the #monitoringlove:** [Learn Sensu Go][1].
 
 Sensu Go is the latest version of Sensu, written in Go and designed to be more portable, easier and faster to deploy, and (even more) friendly to containerized and ephemeral environments.
-But above all, it was designed to support the features and functionality you’ve come to know and love about Sensu.
+For enterprise-class features and support for Sensu Go, try [Sensu Enterprise][28].
```
Comment (bananabele) [inline content/sensu-go/5.2/_index.md]: I'm wondering if we need "for Sensu Go" in this sentence -- it kind of sounds like Sensu Go is what's being supported. With the paragraph above, I think it's clear we're talking about Sensu Go. Thoughts?

File: content/sensu-go/5.2/getting-started/enterprise.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- [Contact the Sensu sales team](https://sensu.io/sales/)
+- [Log in to the Sensu account portal][2]
+- [Contact Sensu support](https://account.sensu.io/support)
+
+### Get started with Sensu Enterprise
+
+If you haven't already, [install the Sensu Go backend, agent, and sensuctl tool](../../installation/install-sensu) and [configure sensuctl](../../sensuctl/reference/#first-time-setup).
+
+Log in to your Sensu account at [account.sensu.io](https://account.sensu.io/) and download your license file.
+Then activate your license using sensuctl.
```
Comment (bananabele) [inline content/sensu-go/5.2/getting-started/enterprise.md]: either remove "then" at the start of the sentence or connect to the preceding sentence: "download your license file, then activate..."

File: content/sensu-go/5.2/getting-started/enterprise.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    parent: getting-started
+---
+
+Sensu Go offers license-activated features designed for enterprises, like LDAP authentication and enterprise-class support.
+[Contact the Sensu sales team][1] for a personalized demo.
+
+**Enterprise-only features in Sensu Go:**
+
+- [LDAP authentication](../../installation/auth)
+- Incident automation using the [Sensu ServiceNow handler](https://bonsai.sensu.io/assets/portertech/sensu-servicenow-handler)
```
Comment (bananabele) [inline content/sensu-go/5.2/getting-started/enterprise.md]: is "incident automation" the standard terminology? to me it sounds like you're automating the incidents themselves 😬 but ignore this comment if this is standard

File: content/sensu-go/5.2/installation/install-sensu.md [hunk 4]
```diff
@@ -360,6 +360,26 @@ sensuctl config set-namespace default
 
 See the [sensuctl reference][4] for more information about using sensuctl.
 
+### 3. Activate enterprise-only features
+
+Sensu Enterprise offers features designed for enterprises, like LDAP authentication and enterprise-class support.
+To learn more about Sensu Enterprise, [contact the Sensu sales team](http://sensu.io.sales) to schedule a demo.
```
Comment (bananabele) [inline content/sensu-go/5.2/installation/install-sensu.md]: this URL should be /sales

File: content/sensu-go/5.2/installation/install-sensu.md [hunk 5]
```diff
@@ -360,6 +360,26 @@ sensuctl config set-namespace default
...
+
+If you already have a Sensu Enterprise account, log in to your Sensu account at [account.sensu.io](https://account.sensu.io/) and download your license file.
+Then apply your license using sensuctl.
```
Comment (bananabele) [inline content/sensu-go/5.2/installation/install-sensu.md]: similar comment to above -- either remove "then" or connect to the preceding sentence

**Comment by bananabele** [comment]
@apaskulin this is looking great! a few minor comments within -- lmk if you'd like to discuss.

File: content/sensu-go/5.2/getting-started/enterprise.md (hunk 3 above)
Comment (apaskulin) [reply content/sensu-go/5.2/getting-started/enterprise.md]: No, it's not standard. My idea here was to add something that indicates why you'd want to use a ServiceNow handler: that it allows Sensu to create and update incidents automatically based on your monitoring events. Let me know what you think. We can always just shorten this to `- Sensu ServiceNow event handler`.

File: content/sensu-go/5.2/_index.md (hunk 1 above)
Comment (apaskulin) [reply content/sensu-go/5.2/_index.md]: > "it kind of sounds like Sensu Go is what's being supported."

This was my intention, but it seems like it's creating some confusion. Let's chat about this synchronously if you have time this week.

File: content/sensu-go/5.2/getting-started/enterprise.md (hunk 3 above)
Comment (bananabele) [reply content/sensu-go/5.2/getting-started/enterprise.md]: +1 for the description on what you can do with it! would it be too wordy to go with the longer description? Maybe tweaking the format to "Sensu ServiceNow event handler: use Sensu to create and update incidents automatically based on your monitoring events" ?

File: content/sensu-go/5.2/_index.md (hunk 1 above)
Comment (bananabele) [reply content/sensu-go/5.2/_index.md]: maybe it's as simple as adding "installation" as in, "your Sensu Go installation" ? upon re-reading it's less confusing (my first read saw it as support=compatibility). either way, happy to discuss

Comment (apaskulin) [reply content/sensu-go/5.2/_index.md]: Options:
1. For enterprise-class support, LDAP authentication, and support for industry-standard tools, try Sensu Enterprise.
2. For enterprise-class features and support for your Sensu Go installation, try Sensu Enterprise.

Comment (bananabele) [reply content/sensu-go/5.2/_index.md]: let's do 2 -- I like the call out to LDAP in 1, but I'm wondering if that opens the door to a very long list once we add more features.

**Comment by apaskulin** [comment]
Thanks, @bananabele! This is ready for another review.

File: content/sensu-go/5.2/getting-started/faq.md [hunk 6]
```diff
@@ -29,7 +29,8 @@ See the [hardware requirements guide][5] for minimum and recommended hardware to
 
 > Is there an enterprise version of Sensu Go?
 
-Yes! Sensu Go for enterprise is coming soon. Check out this [Sensu blog post](https://blog.sensu.io/announcing-the-sensu-roadmap-sensu-go-release-date) for more information about Sensu Go for enterprise.
+[Yes!](https://blog.sensu.io/enterprise-features-in-sensu-go) Sensu Inc. offers support packages for Sensu Go as well as license-activated features designed for enterprises, like LDAP authentication and support for industry-standard tools.
```
Comment (bananabele) [inline content/sensu-go/5.2/getting-started/faq.md]: extremely minor nit -- the feature elaboration after "enterprises" sounds like we're describing the enterprises. is there a way to shuffle?
